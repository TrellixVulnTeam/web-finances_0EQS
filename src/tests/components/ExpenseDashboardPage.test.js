import React from 'react'
import { shallow } from 'enzyme'
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage'

test('should render ExpenseDashboardPage with all attributes', () => {
    const wrapper = shallow(<ExpenseDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})

test('should render ExpenseDashboardPage', () => {
   const wrapper = shallow(<ExpenseDashboardPage />)
   expect(wrapper).toMatchSnapshot() 
})
