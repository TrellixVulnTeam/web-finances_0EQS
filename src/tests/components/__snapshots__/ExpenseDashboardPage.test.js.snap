// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage 1`] = `
<div>
  <[object Object] />
  <[object Object] />
</div>
`;

exports[`should render ExpenseDashboardPage with all attributes 1`] = `
<div>
  <[object Object] />
  <[object Object] />
</div>
`;
